docs/source/release/v6.4.0/direct_geometry.rst

```diff
@@ -59,18 +59,17 @@ MSlice
 New Features
 ############
 
-- Various documentation updates
-- MSlice now available as a noarch conda package
-- Update to matplotlib 3.5
-- Add ability to change slice plot font sizes using quick options
-- Changed default cut algorithm to Integration
+- The documentation has been improved.
+- The programme is now available as a noarch conda package.
```

Comment:
I think perhaps Conda rather than conda?

Reply:
They write it as lowercase on their homepage when it's not at the start of a sentence, so I'll leave this one as-is